src/Run/Setup.cs

```diff
@@ -57,6 +93,35 @@ private string ParseSettingValue(string inputValue)
             return value;
         }
 
+        private void ParseRunToolSettings(string commandSelectedByUser = null)
+        {
+            Setting tempSetting;
+            Command tempCommand;
+            foreach(var toolSetting in ToolSettings.Keys.Select(k => k.ToString()).ToArray())
+            {
+                string tempValue = null;
+                // Attempt to get run tool setting value from parameters
+                SettingParameters.TryGetValue(toolSetting, out tempValue);
+                ToolSettings[toolSetting] = tempValue;
+
+                // Attempt to get run tool setting value from command section
+                if (string.IsNullOrEmpty(ToolSettings[toolSetting]) && commandSelectedByUser != null)
+                {
+                    if (Commands.TryGetValue(commandSelectedByUser, out tempCommand))
+                    {
+                        tempCommand.DefaultValues.Settings.TryGetValue(toolSetting, out tempValue);
+                        ToolSettings[toolSetting] = tempValue;
+                    }
+                }
+                // Attempt to get run tool setting value from settings section
+                if (string.IsNullOrEmpty(ToolSettings[toolSetting]))
+                {
+                    Settings.TryGetValue(toolSetting, out tempSetting);
```

Comment:
I'm wondering about this behavior. Looking at the value in Settings means that we are always going to use the default value of the Setting that in the majority of cases will always be turned off (with false or whatever).
But, if the user wants that particular setting to be used and it is not part of a command, then the user should specified that he wants it.
For example, right now the `runquiet` option is only in the `produces` command. I want to use it during sync,
so I should do: `sync -runQuiet` right?

All of this to say that maybe we shouldn't look at the value in the `Settings` dictionary, we should look for it in the `SettingParameters` dictionary https://github.com/dotnet/buildtools/pull/980/files#diff-6f4e4ec63e7b18f5f61e1d0ba5c10087L17

What do you think?